src/pages/cli/project/override.mdx

```diff
@@ -33,4 +33,40 @@ You can override the following project-level resources that Amplify generates:
 |Amplify-generated resource|Description|
 |-|-|
 |[authRole](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html)|The IAM role for authenticated access to your app backend|
-|[unauthRole](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html)|The IAM role for authenticated or guest access to your app backend|
\ No newline at end of file
+|[unauthRole](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html)|The IAM role for authenticated or guest access to your app backend|
+
+
+## Customize unauthRole policies
+
+For example, to use the project override to update the unauthRole policy for S3, use the following:
+
+```ts
+import { AmplifyRootStackTemplate } from "@aws-amplify/cli-extensibility-helper";
+
+export function override(resources: AmplifyRootStackTemplate) {
+  const unauthRole = resources.unauthRole;
+
+  const basePolicies = Array.isArray(unauthRole.policies)
+    ? unauthRole.policies
+    : [unauthRole.policies];
+
+  unauthRole.policies = [
+    ...basePolicies,
+    {
+      policyName: "amplify-permissions-custom-resources",
+      policyDocument: {
+        Version: "2012-10-17",
+        Statement: [
+          {
+            Resource: "<ARN of resource>",
+            Action: ["s3:PutObject", "s3:GetObject", "s3:DeleteObject"],
+            Effect: "Allow",
+          },
+        ],
+      },
+    },
+  ];
```

Comment:
nit: given other examples don't have semi-colons, we should probably skip it here.